aiozipkin/helpers.py

```diff
@@ -85,6 +89,26 @@ def make_headers(context: TraceContext) -> Headers:
     return headers
 
 
+def make_single_header(context: TraceContext) -> Headers:
+    """Creates dict with zipkin single header format.
+    """
+    # b3={TraceId}-{SpanId}-{SamplingState}-{ParentSpanId}
+    c = context
+
+    # encode sampled flag
+    sampled = 'd' if c.sampled is None else str(int(c.sampled))
```

Comment:
sampled == '1' debug == 'd' is this backwards? (PS unsampled is '0'). None is absence of the field itself

Reply:
Yes, it looks like this would explicitly enable debug sampling.
Maybe this should be
```python
sampled = 'd' if not c.sampled and c.debug else str(int(c.sampled))
```
getting rid of the additional `if c.debug` 
and below
```python
def parse_single_header():
   snip
    if len(parts) >= 3:
       if parts[2] in ('1', '0'):
            sampled = bool(int(parts[2]))
       elif 'd' in parts[2]:
            debug = True
```

Reply:
sounds correct. note that debug is implicitly also sampled. you can think of it as a boosted sample decision.

Reply:
ah I confused debut with deferred from multi header propagation, will fix in separate PR.

Reply:
Here is PR https://github.com/aio-libs/aiozipkin/pull/191